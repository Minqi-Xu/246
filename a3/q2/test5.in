f 2
r
f 8
r
f 4
l
f 2
c
t
m
