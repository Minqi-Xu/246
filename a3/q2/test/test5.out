Current position: (10,-2), Facing East
16
12
